pages/phases/phone-assessment-interviews/reviewing-sme-assessments.md

```diff
@@ -16,6 +16,10 @@ Verify that every core competency has a rating of Does Not Meet, Meets, or Excee
 - Verify that the summary interview analysis is complete (with at least a few sentences).
 - Confirm the SME correctly addressed any initial issues you may have reported.
 
+### Calculating an Applicant's Final Assessment Rating
+
+After SMEs have addressed your feedback for their interview assessment forms, calculate the applicant's final assessment rating based on the SME's proficiency determination using the <a href="{{site.baseurl}}/toolkit/phone-assessment-interviews/interview-assessment-ratings/">Interview Assessment Rating Guide</a>.
```

Comment:
calculate and save?

Reply:
@bernars-usa this step got slightly more confusing for HR with martha's revision to the interview template (which i think was worth it). Now HR doesnt clearly see if the SME said the applicant meets the requirements or not. Now HR will jsut see if the SME checked the proficiency level that is required for the applicant to be minimally qualified. If they did, HR needs to check off "meets" for that proficiency and THEN calculate the rating. make sense?

Reply:
Hmm. I think we need to try to make this easier for HR, too. I'll document it the way it is for now, but the point was to clearly see what "Required" (or maybe "Minimum Required") is.